### ISSUE TITLE
Added reactions 

### ISSUE THREAD

**Issue by SamarthKadam** [SamarthKadam/ChatBox#295]
Add Reactions to Messages. You can react to messages. Just provide 4 limited reaction options

**Comment by Ansh101112** [comment]
> Add Reactions to Messages. You can react to messages. Just provide 4 limited reaction options

I would like to work on this issue please assign me this issue

**Comment by SamarthKadam** [comment]
Anybody can ping here to get assigned

**Comment by ojaswichopra** [comment]
I can work on this

**Comment by ojaswichopra** [comment]
Please assign this issue to me

**Comment by SamarthKadam** [comment]
@ojaswichopra  your good to go

**Comment by ojaswichopra** [comment]
@SamarthKadam I have some urgent commitments so m unable to do this. M sorry I had to remove the assignment.

**Comment by BHS-Harish** [comment]
Hey @SamarthKadam 

I'm interested to work on this issue
Please assign it to me

**Comment by SamarthKadam** [comment]
Yeah try to keep it simple. Try to add only 4 most used reactions

**Comment by Prince2301p** [comment]
Is there an existing issue for this?

I have searched the existing issues
New Tool Description
Slite can help you keep up with your work. This app serves as a knowledge base where you can store your business’s collective brainpower.

Use Case
Slite can help you keep up with your work. This app serves as a knowledge base where you can store your business’s collective brainpower.

Benefits
No response

Add ScreenShots
No response

Priority
High

Record

I have read the Contributing Guidelines

I'm a GSSOC'24 contributor

I have starred the repository

**Comment by billu-beep** [comment]
Hi @SamarthKadam! I'd love to work on this issue. 
I plan to add 4 most used emoji reactions (👍 ❤️ 😂 😮) 
with a simple hover UI. Please assign it to me! 🙏
